src/main/java/tests/OperationHandlersTest.java

```diff
@@ -0,0 +1,65 @@
+package tests;
+
+import core.basesyntax.*;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class OperationHandlersTest {
+    @Test
+    public void testBalanceOperation() {
+        OperationHandler handler = new BalanceOperation();
```

Comment:
Direct casting to `BalanceOperation` is a sign of poor design. You should use the `OperationHandler` interface without casting, ensuring that the code adheres to the Liskov Substitution Principle part of SOLID.